hadoop/src/main/java/org/apache/carbondata/hadoop/streaming/CarbonStreamRecordReader.java

```diff
@@ -342,7 +349,11 @@ private boolean nextRow() throws IOException {
             input.nextRow();
             scanMore = false;
           } else {
-            readRowFromStream();
+            if (useRawRow) {
+              readRawRowFromStream();
```

Comment:
add comment to describe this is for `handoff`, which does not require decode raw row